ansible_rulebook/validators.py

```diff
@@ -0,0 +1,31 @@
+import json
+import logging
+import os
+from typing import ClassVar, Dict, List
+
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
+
+DEFAULT_RULES_SCHEMA = "../schema/ruleset_schema.json"
```

Comment:
I think the naming is misleading. This variable should be named `DEFAULT_RULEBOOK_SCHEMA` as well as the file should be named `rulebook_schema.json`